### DISCUSSION TITLE
How to export the home manager config from my own nixvim flake ?

### DISCUSSION THREAD

**Discussion by elythh** [nix-community/nixvim#1439] [Q&A] [answered]
Hey, I'm still new to NixOS/Home-manager and I'd like to know of somebody here could here me out.

I have my own custom [Nixvim config as flake](https://github.com/elythh/nixvim). For now it exports as nvim package but I'd also like to export the home-manager config that way I can import it in my NixOS flake and customize it there too. 

Does anyone know how can I modify my flake to do so ? Thanks a lot for your help

**Comment by MattSturgeon** [comment, marked as answer]
You don't need to modify your nixvim flake. Instead you can make use of [the `nixvimExtend` function](https://nix-community.github.io/nixvim/modules/standalone.html#extending-an-existing-configuration) to _extend_ your standalone nixvim in your other config.

**Reply by elythh** [reply]
That looks like exactly what I was looking for, thank you !
